### PR TITLE
Test case for issue 5567

### PR DESCRIPTION
**Pull request by timvandermeij**
Thanks to @eldonkijote for providing us with the test image!

### CODE REVIEW

**Comment by CodingFabian** [comment]
looks like both bots need a kick @yurydelendik

File: test/test_manifest.json [hunk 1]
```diff
@@ -1967,5 +1967,11 @@
        "rounds": 1,
        "type": "eq",
        "about": "Free image obtained from www.unsplash.com"
+    },
+    {  "id": "issue5567.pdf",
```
Comment (Snuffleupagus) [inline test/test_manifest.json]: Nit: You don't really need ".pdf" in the `id`, since this is just the unique name of the test.

Comment (timvandermeij) [inline test/test_manifest.json]: Fixed.

**Comment by timvandermeij** [comment]
Both bots have the Firefox process still running again. Let's retrigger testing after @yurydelendik kicked the bots.

**Pull request merged**